triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -293,34 +338,63 @@ object Server {
       },
     )
 
+    // The server binding is a future that on completion will be piped
+    // to a message to this actor.
     val serverBinding = Http().bindAndHandle(Route.handlerFlow(route), host, port)
     ctx.pipeToSelf(serverBinding) {
       case Success(binding) => Started(binding)
       case Failure(ex) => StartFailed(ex)
     }
 
+    // The server running server.
     def running(binding: ServerBinding): Behavior[Message] =
       Behaviors
         .receiveMessage[Message] {
-          case StartFailed(_) => Behaviors.unhandled
-          case Started(_) => Behaviors.unhandled
+          case TriggerStarting(uuid, jwt, runner) =>
```

Comment:
Is there a timeout on the time between starting and started? If not, should there be one? (not in this PR)

Reply:
There currently isn't. Not a bad idea.